content/800-data-platform/050-accounts.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: 'Accounts'
+metaTitle: 'Accounts'
+metaDescription: 'To use the Prisma Data Platform, you can sign up with your GitHub account. If you already have an existing GitHub login session, '
+tocDepth: 3
+---
+<TopBlock>
+
+To start using the Prisma Data Platform, you need to sign up with your GitHub account. When you do so, you authorize the Prisma GitHub app to read the list of repositories (to which you have granted access), and to read and create new repositories for new projects.
+
+In addition to your GitHub, you can also use your accounts for [PlanetScale](https://planetscale.com/) and [Heroku](https://www.heroku.com/) when you want to use either platform to quickly provision a new database for your projects.
+
+Any PlanetScale or Heroku account that you use is a _linked account_ in the Prisma Data Platform. At any time, you can link only one PlanetScale and only one Heroku account. However, you can unlink any currently linked accounts and use others to quickly provision databases.
```

Comment:
```suggestion
Any PlanetScale or Heroku account that you use with the Prisma Data Platform is a _linked account_. At any time, you can link only one PlanetScale and only one Heroku account. However, you can unlink any currently linked accounts and use others to quickly provision databases.
```

I thought this phrasing makes it more clear that we're talking about accounts that were used in PDP, and not just some other PS/Heroku account you have open in your browser session that you used parallel to using PDP.